package.json

```diff
@@ -34,19 +34,19 @@
     "node": ">=8.10.0"
   },
   "devDependencies": {
-    "@iopipe/core": "1.11.0",
     "@iopipe/scripts": "^1.4.1",
     "aws-lambda-mock-context": "^3.1.1",
     "lodash": "^4.17.4",
+    "nock": "^9.4.1",
     "pre-commit": "^1.2.2"
   },
   "dependencies": {
+    "@iopipe/core": "^1.13",
     "archiver": "^2.1.1",
-    "lodash.get": "^4.4.2"
-  },
-  "peerDependencies": {
-    "@iopipe/core": "^1.5.0"
+    "lodash.get": "^4.4.2",
+    "simple-get": "^3.0.2"
   },
+  "peerDependencies": {},
```

Comment:
do you mean to put `@iopipe/core` here?

Reply:
No, it's a true dep now as we require code to actually use the lib.